fitbert_test_data/get_gap.py

```diff
@@ -0,0 +1,105 @@
+import re
+
+import numpy as np
+import pyinflect
+import spacy
+
+from get_options import get_related_options, get_verb_options
+from get_phrase import get_noun_phrase, get_verb_phrase
+
+
+def mask_sentence(sentence_span, start, end, mask):
+    masked_sentence_tok = [token.text for token in sentence_span]
+    masked_sentence_tok[start] = mask
+    masked_sentence_tok[start + 1 : end] = ""
+    masked_sentence = re.sub(r"\s{2, }", " ", " ".join(masked_sentence_tok)).strip()
+    return masked_sentence
+
+
+def get_verb_gap(sentence_span, mask_token="***mask***"):
+
+    verb_phrases = get_verb_phrase(sentence_span)
+
+    if len(verb_phrases):
+        chosen_verb_phrase = sorted(
+            verb_phrases, key=lambda x: x["end"] - x["start"], reverse=True
+        )[0]
+
+        if len(chosen_verb_phrase["main_verb"]):
+            main_verb = chosen_verb_phrase["main_verb"][0]
+            answer = chosen_verb_phrase["verb_phrase"].text.lower()
+
+            adverbs = chosen_verb_phrase["adverbs"]
+            verb_phrase_inffix = " ".join([a.text for a in adverbs])
+
+            start = chosen_verb_phrase["start"]
+            end = chosen_verb_phrase["end"]
+
+            options = get_verb_options(main_verb, verb_phrase_inffix)
+            options_len = [1 / len(o) for o in options]
```

Comment:
Can we add some checks around here to make 100% sure we aren't dividing by 0?

Reply:
There is no chance to get division by zero here. 
See: implementation of conjugate. I am checking whether VB form exists. If VB exist then there are no options of len 0 possible.